formuledevoyage_card.php

```diff
@@ -653,7 +653,23 @@
 	$diroutput = $conf->clienjoyholidays->dir_output;
 	$trackid = 'formuledevoyage'.$object->id;
 
+
+	// Adds default receivers to be the "Voyageurs"
+	if (!GETPOSTISSET('receiver')){
+		$Tcontacts = $object->liste_contact('', 'external', '0', 'VOYAGER');
+		if ($Tcontacts == -1) {
+			$this->error = $langs->trans("ErrorGettingContact");
+			setEventMessages('', [$this->error], 'errors');
+			dol_syslog(get_class($this)."::valid ".$this->error, LOG_ERR);
```

Comment:
tu n'es pas dans une classe pourquoi utiliser $this, passe par une variable, ça fait pas une erreur php ça d'ailleurs ?

Reply:
et pourquoi ::valid ?